mifosng-android/src/main/java/com/mifos/mifosxdroid/online/collectionsheetindividualdetails/IndividualCollectionSheetDetailsFragment.java

```diff
@@ -0,0 +1,219 @@
+        setToolbarTitle(getStringMessage(R.string.individual_collection_sheet));
+        sheetsAdapter = new IndividualCollectionSheetDetailsAdapter(getContext(), this);
+        presenter.attachView(this);
+        this.payload = ((GenerateCollectionSheetActivity) getActivity()).getPayload();
+        showCollectionSheetViews(sheet);
+        return rootView;
+    }
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        inflater.inflate(R.menu.menu_individual_collectionsheet, menu);
+        super.onCreateOptionsMenu(menu, inflater);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.item_submit_sheet:
+                submitSheet();
+                return true;
+        }
+        return super.onOptionsItemSelected(item);
+    }
+
+    private void showCollectionSheetViews(IndividualCollectionSheet sheet) {
+        paymentTypeList = presenter.filterPaymentTypeOptions(sheet.getPaymentTypeOptions());
+        loansAndClientNames = presenter.filterLoanAndClientNames(sheet.getClients());
+
+        //Initialize payload's BulkRepaymentTransactions array with default values.
+        //The changes made (if any) will be updated by the interface 'OnRetrieveSheetItemData'
+
+        //methods.
+        if (payload == null) {
+            payload = new IndividualCollectionSheetPayload();
+            for (LoanAndClientName loanAndClientName :
+                    presenter.filterLoanAndClientNames(sheet.getClients())) {
+                LoanCollectionSheet loanCollectionSheet = loanAndClientName.getLoan();
+                payload.getBulkRepaymentTransactions().add(new BulkRepaymentTransactions(
+                        loanCollectionSheet.getLoanId(),
+                        loanCollectionSheet.getChargesDue() == null ?
+                                loanCollectionSheet.getTotalDue() :
+                                loanCollectionSheet.getTotalDue() +
+                                        loanCollectionSheet.getChargesDue()));
+            }
+        }
+        LinearLayoutManager layoutManager = new LinearLayoutManager(getContext());
+        DividerItemDecoration dividerItemDecoration = new DividerItemDecoration(
+                recyclerSheets.getContext(), layoutManager.getOrientation());
+        recyclerSheets.setLayoutManager(layoutManager);
+        recyclerSheets.addItemDecoration(dividerItemDecoration);
+        recyclerSheets.setAdapter(sheetsAdapter);
+
+        sheetsAdapter.setSheetItemClickListener(this);
+        sheetsAdapter.setLoans(loansAndClientNames);
+        sheetsAdapter.setPaymentTypeList(paymentTypeList);
+        sheetsAdapter.setPaymentTypeOptionsList(sheet.getPaymentTypeOptions());
+        sheetsAdapter.notifyDataSetChanged();
+    }
+
+    @Override
+    public void showSuccess() {
+        Toast.makeText(getContext(), getStringMessage(R.string.collectionsheet_submit_success),
```

Comment:
You are using `Toaster` everywhere in this class except for this message. For the sake of consistency, use the same class here as well.